src/platform_impl/android/mod.rs

```diff
@@ -673,16 +673,12 @@ impl WindowId {
     pub const fn dummy() -> Self {
         WindowId
     }
-}
 
-impl From<WindowId> for u64 {
-    fn from(_: WindowId) -> Self {
+    pub fn into_raw(self) -> u64 {
         0
     }
-}
 
-impl From<u64> for WindowId {
-    fn from(_: u64) -> Self {
+    pub fn from_raw(_id: u64) -> Self {
```

Comment:
Can we mark them as `const`?

Reply:
`from_raw()` yes, but `into_raw()` not unfortunately, MacOS does pointer casting here.

Reply:
Hmm, we could just make `WindowId` contain `u64` instead of a pointer or whatever it's doing now?

Reply:
Feel free to push to this branch if you want to do it in this PR!

Reply:
Done now